kits/sparktrace/apps/.env.example

```
@@ -0,0 +1,55 @@
+# assumed role / short-lived credentials over long-lived keys.
+AWS_ACCESS_KEY_ID="AWS_ACCESS_KEY_ID"
+AWS_SECRET_ACCESS_KEY="AWS_SECRET_ACCESS_KEY"
+AWS_SESSION_TOKEN=""
+AWS_REGION="us-east-1"
+
+# ── Athena ──────────────────────────────────────────────────
+# ATHENA_WORKGROUP should be scoped to a READ-ONLY IAM policy and
+# have a bytesScannedCutoffPerQuery configured — this is the
+# infrastructure-level backstop behind the deterministic query
+# guard (apps/lib/safety/query-guard.ts) and the constitution.
+ATHENA_WORKGROUP="primary"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)